src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -77,7 +77,8 @@ export class DownloadCLIClientComponent extends Base implements OnInit {
     this.textDataRequirements = `
 ### Setup Command Line Interface
 ------------------------------
-Setup our Dockstore CLI application to start launching workflows from the command line.
+Setup our Dockstore CLI application in order to test workflows from the command line for [local development](https://docs.dockstore.org/en/stable/launch-with/launch.html#dockstore-cli), [validate .dockstore.yml files](https://docs.dockstore.org/en/stable/advanced-topics/dockstore-cli/yaml-command-line-validator-tool.html) for registering tools and workflows,
```

Comment:
Was like this, but "Setup" is one word if it's a noun, two if it's a verb. I had to look that up to confirm :)

Reply:
For the docs links, you should use `DOCKSTORE.DOCUMENTATION_URL` instead of hard-coding.